specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json

```diff
@@ -1889,6 +2158,109 @@
       ],
       "description": "The List disk encryption set operation response."
     },
+    "PrivateLinkServiceConnectionState":{
+      "properties":{
+        "actionsRequired": {
+          "type": "string",
```

Comment:
Is this not an enum?

Reply:
yes also its a readonly so we mayn't  need to expose the enum